report_thesis/src/sections/results/visual_analysis.tex

```diff
@@ -1,51 +1,62 @@
-\subsubsection{Visual Analysis of \ce{SiO_2} Distribution in Folds}
-This section aims to provide a detailed visual and statistical analysis of the \ce{SiO_2} concentration distribution across different folds used in our custom k-fold data partitioning procedure.
-The purpose is to demonstrate the effectiveness of our strategy in maintaining balanced and representative datasets, which is crucial for developing robust predictive models.
-
-We focus specifically on \ce{SiO_2} as a representative example to illustrate our methodology.
-Similar analyses were conducted for other oxides; however, the plots and figures for these are omitted here for brevity.
+\subsection{Visual and Statistical Analysis of \ce{SiO_2} Distribution in Partitioned Data}\label{sec:visual_analysis}
+This section provides a detailed visualization and statistical analysis of the \ce{SiO_2} concentration distribution across data partitions, following the customized k-fold data partitioning procedure described in Section~\ref{subsec:validation_testing_procedures}.
+This analysis is conducted to validate the consistency of our data partitioning method and to provide a visual understanding of the resulting data distribution.
+The analysis focuses on \ce{SiO_2} as a representative example.
+Similar analyses have been conducted for other oxides, and the resulting plots are shown in Appendix~\ref{subsec:cv_plots}.
+
+As discussed in Section~\ref{subsec:validation_testing_procedures}, it is crucial to determine an optimal value of $p$ for the data partitioning algorithm that minimizes the number of extreme values in the test set while still maintaining its representativeness.
+Our general approach is to select the lowest value of $p$ that ensures the test set contains no extreme values while maximizing its general representativeness.
+This approach helps avoid overestimating the model's performance due to extreme values while keeping the test set reflective of the majority of the data distribution.
```

Comment:
It's important to keep the term `general representativeness` because we define it in testing & validation.

> By general representativeness, we mean ensuring that the test set reflects the general distribution of the dataset without being skewed by extreme values.

It does get wordy, but this exact sentence structure is what we support. The words that are being removed aren't filler, but IMO necessary precision and nuance.

```suggestion
Our approach is to select the lowest $p$ that excludes extreme values from the test set while maximizing its general representativeness.
This approach helps avoid overestimating the model's performance due to extreme values while keeping the test set reflective of the majority of the data distribution.
```